content/people/damiri.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Delaram Amiri
+role: student-phd
+status: Ph.D. Candidate
+yearjoined: 2015
+portrait: damiri.jpg
+badge: damiri-badge.jpg
+interests:
+    - Healthcare IoT
+    - Sensor Control
+    - Optimization
+    - Edge Computing
+education:
+    - degree: M.S. in Electrical Engineering, Communications
+      institution: Indiana University–Purdue University Indianapolis, U.S.
+      year: 2015
+    - degree: B.S. in Electrical Engineering, Communications
+      institution: Shiraz University, Iran
+      year: 2013
+email: damiri@uci.edu
+websites:
+    - title: LinkedIn
+      url: https://www.linkedin.com/in/delaramamiri
+bibliography:
+  - title: Journal Articles 
+    items: 
+    - delaram1 
+    - delaram2 
+    - delaram3 
+    - delaram4
+  - title: Conference Proceedings 
+    items: 
+    - delaram6 
+    - delaram7
+  - title: M.S. Dissertation
```

Comment:
“M.S. Thesis” is more conventional than “M.S. Dissertation“—let’s use
the former instead of the latter.